### PR TITLE
Solution Migration Changes for GCPIAM

### PR DESCRIPTION
**Pull request by v-mchatla**
Change(s):

See guidance below
Reason for Change(s):
This PR is part of template spec solution migration for solution Google Cloud Platform IAM.

See guidance below
Version Updated:
2.0.0

See guidance below
Testing Completed:
The link for test results is

https://microsoftapc.sharepoint.com/:w:/r/teams/SentinelCATTeam/_layouts/15/Doc.aspx?sourcedoc=%7B3A0CB2A9-5F8D-459A-B3AF-2E0A8D772042%7D&file=Sentinel%20Solution%20Migration%20-TemplateSpec%20-%20GCPIAM.docx&action=default&mobileredirect=true&share=IQGpsgw6jV-aRbOvLgqNdyBCAZOjzfMOnG6WWFhpVZgIJxs

See guidance below
This is a 3P solution.

### CODE REVIEW

File: Solutions/GoogleCloudPlatformIAM/Data/Solution_GoogleCloudPlatformIAM.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	  "Hunting Queries/GCPIAMTopServiceAccountsFailedActions.yaml",
+	  "Hunting Queries/GCPIAMTopSrcIpAddrFailedActions.yaml"
+    ],
+    "BasePath": "C:\\GitHub\\Azure-Sentinel\\Solutions\\GoogleCloudPlatformIAM",
+	"Version": "2.0.0",
+	"Metadata": "SolutionMetadata.json",
+	"TemplateSpec": true,
+	"Is1Pconnector": false
+  }
+
```
Comment (v-atulyadav) [inline Solutions/GoogleCloudPlatformIAM/Data/Solution_GoogleCloudPlatformIAM.json:45]: Hey @v-mchatla , please remove blank row

File: Solutions/GoogleCloudPlatformIAM/SolutionMetadata.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+		"verticals": []
+	},
+	"support": {
+		"name": "Microsoft Corporation",
+		"email": "support@microsoft.com",
+		"tier":"Microsoft",
+		"link":"https://support.microsoft.com"
+	}
+}
+
```
Comment (v-atulyadav) [inline Solutions/GoogleCloudPlatformIAM/SolutionMetadata.json:17]: Hey @v-mchatla, please remove blank rows

**Review by v-atulyadav — COMMENTED** [review]
Hey @v-mchatla, please change text from `Azure Sentinel` to `Microsoft Sentinel`

File: Solutions/GoogleCloudPlatformIAM/Data/Solution_GoogleCloudPlatformIAM.json (hunk 1 above)
Comment (v-mchatla) [reply Solutions/GoogleCloudPlatformIAM/Data/Solution_GoogleCloudPlatformIAM.json]: As suggested, removed the extra space

File: Solutions/GoogleCloudPlatformIAM/SolutionMetadata.json (hunk 2 above)
Comment (v-mchatla) [reply Solutions/GoogleCloudPlatformIAM/SolutionMetadata.json]: As suggested, removed the extra space

**Review by v-atulyadav — APPROVED** [review]
Looks good & approved

**Pull request merged**